extras/userdoc/md/documentation/installation.md

```diff
@@ -137,7 +137,7 @@ The easiest way to install NEST on a Mac is to install it via the Homebrew packa
 
     1.  Add the homebrew/science tap by running 
     
-        `brew tap homebrew/science`
+        `brew tap brewsci/science`
```

Comment:
Hopefully temporary: need to be changed, when https://github.com/Homebrew/homebrew-core/pull/24908 is merged.